### PR TITLE
Upgrade supported PHP versions to PHP ^5.6, ^7.0, and ^8.0

### PR DESCRIPTION
**Pull request by Ayesh**
Related: #52

Let's upgrade to PHP 5.6 :) 
Please note that this patch is necessary for #52 to pass CI, so ideally this PR is merged first. Merging this will result in a merge-conflict for #52, but I can work on a another push that would [fix everything](https://travis-ci.com/github/Ayesh/composer-bin-plugin/builds/160676069).

### CODE REVIEW

File: src/BinCommand.php [hunk 1]
```diff
@@ -22,10 +22,12 @@ protected function configure()
...
             ->setName('bin')
             ->setDescription('Run a command inside a bin namespace')
-            ->setDefinition(array(
+            ->setDefinition(
+                [
                 new InputArgument('namespace', InputArgument::REQUIRED),
                 new InputArgument('args', InputArgument::REQUIRED | InputArgument::IS_ARRAY),
-            ))
+                ]
+            )
```
Comment (theofidry) [inline src/BinCommand.php:30]:
```suggestion
            ->setDefinition([
                new InputArgument('namespace', InputArgument::REQUIRED),
                new InputArgument('args', InputArgument::REQUIRED | InputArgument::IS_ARRAY),
            ])
```

File: tests/BinCommandTest.php [hunk 2]
```diff
@@ -23,16 +23,18 @@ protected function setUp()
...
 
         $this->application = new Application();
-        $this->application->addCommands(array(
+        $this->application->addCommands(
+            [
             new BinCommand(),
             $this->myTestCommand = new MyTestCommand($this),
-        ));
+            ]
+        );
```
Comment (theofidry) [inline tests/BinCommandTest.php:31]:
```suggestion
        $this->application->addCommands([
            new BinCommand(),
            $this->myTestCommand = new MyTestCommand($this),
        ]);
```

File: tests/Fixtures/MyTestCommand.php [hunk 3]
```diff
@@ -20,9 +20,11 @@ public function __construct(\PHPUnit_Framework_Assert $assert)
         $this->assert = $assert;
 
         parent::__construct('mytest');
-        $this->setDefinition(array(
+        $this->setDefinition(
+            [
             new InputOption('myoption', null, InputOption::VALUE_NONE),
-        ));
+            ]
+        );
```
Comment (theofidry) [inline tests/Fixtures/MyTestCommand.php:27]:
```suggestion
        $this->setDefinition([
            new InputOption('myoption', null, InputOption::VALUE_NONE),
        ]);
```

**Comment by Ayesh** [comment]
Thanks for the review @theofidry ; you are right those alignments way out of whack. I replaced 60ce5d3 with a73fbdf that has proper alignments like you suggested. Thank you.

**Comment by theofidry** [comment]
Thanks!

**Pull request merged**